### PR TITLE
Better formatting of multiple dates

### PR DESCRIPTION
**Pull request by brucemiller**
This should handle the scenario in aastex.

### CODE REVIEW

File: lib/LaTeXML/resources/XSLT/LaTeXML-structure-xhtml.xsl [hunk 1]
```diff
@@ -644,23 +646,34 @@
...
+    <xsl:param name="dates" select="ltx:date"/>
+    <xsl:if test="$dates">
+      <xsl:text>&#x0A;</xsl:text>
+      <!-- Originally, html5 seemed to suggest we might use h2 here, but that is retracted-->
+      <xsl:element name="div" namespace="{$html_ns}">
+        <xsl:attribute name="class">ltx_dates</xsl:attribute>
+        <xsl:apply-templates select="." mode="begin">
+          <xsl:with-param name="context" select="$context"/>
+        </xsl:apply-templates>
+        <xsl:apply-templates select="$dates" mode="inintitle">
```
Comment (dginev) [inline lib/LaTeXML/resources/XSLT/LaTeXML-structure-xhtml.xsl:660]: typo? `inintitle` -> `intitle`

File: lib/LaTeXML/resources/XSLT/LaTeXML-structure-xhtml.xsl [hunk 2]
```diff
@@ -644,23 +646,34 @@
...
+          <xsl:with-param name="context" select="$context"/>
+        </xsl:apply-templates>
+        <xsl:apply-templates select="." mode="end">
+          <xsl:with-param name="context" select="$context"/>
+        </xsl:apply-templates>
+      </xsl:element>
+    </xsl:if>
+  </xsl:template>
+
+  <xsl:template match="ltx:date" mode="inintitle">
```
Comment (dginev) [inline lib/LaTeXML/resources/XSLT/LaTeXML-structure-xhtml.xsl:670]: same typo? `inintitle` -> `intitle`

**Review by dginev — CHANGES_REQUESTED** [review]

File: lib/LaTeXML/resources/XSLT/LaTeXML-structure-xhtml.xsl [hunk 3]
```diff
@@ -644,23 +646,34 @@
...
+    <xsl:param name="context"/>
+    <xsl:if test="@name"><xsl:value-of select="@name"/><xsl:text>: </xsl:text></xsl:if>
```
Comment (dginev) [inline lib/LaTeXML/resources/XSLT/LaTeXML-structure-xhtml.xsl:672]: This is a cool approach. I tested locally and noticed that `@name` actually has no attribute to select against in the aastex63 example. The attribute for `ltx:date` that is currently in use is role, and switching this test to `@role` got me the visuals I expected locally. Was there a reason for `@name`?

**Comment by brucemiller** [comment]
@Name was more intentional (most metadata has a role, but you don't 
always want that textual prefix), and name is already used in a few
cases.  A patch coming shortly after you rebase adds name= to aas_support :>

File: lib/LaTeXML/resources/XSLT/LaTeXML-structure-xhtml.xsl (hunk 1 above)
Comment (brucemiller) [reply lib/LaTeXML/resources/XSLT/LaTeXML-structure-xhtml.xsl]: Ah, yeah. Actually was intentional during debugging, but is probably just confusing now.  I've fixed it.

**Comment by dginev** [comment]
LGTM! Didn't quite catch which branch should get the `name` patch, and who's adding it, but this PR is definitely a nice improvement already :+1:

**Review by dginev — APPROVED** [review]

**Pull request merged**